### PR TITLE
Config Routes

### PR DESCRIPTION
**Pull request by shaqayq**
In this part of the project :
create `user `and `user_posts ` URLs and for each URL  add:
- A route.
- An action in the correct controller.
- A view file.

### CODE REVIEW

File: app/views/posts/index.html.rb [hunk 1]
```diff
@@ -0,0 +1 @@
+<h1>Post data will load</h1>
```
Comment (thecodechaser) [inline app/views/posts/index.html.rb:1]: - [ ] Please also add a view file `show.html.erb` for the post-show action.

File: app/views/users/index.html.rb [hunk 2]
```diff
@@ -0,0 +1 @@
+<h1>User data will load</h1>
```
Comment (thecodechaser) [inline app/views/users/index.html.rb:1]: - [ ] Please also add a view file `show.html.erb` for user-show action.

File: config/routes.rb [hunk 3]
```diff
@@ -3,4 +3,10 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
```
Comment (thecodechaser) [inline config/routes.rb:6]: - [ ] Please add users#index as the root route as on the wireframe. You can consider using the below code.
```Ruby
root "users#index"

File: config/routes.rb [hunk 4]
```diff
@@ -3,4 +3,10 @@
...
+  get "/users", to: "users#index"
+  get "/users/:id", to: "users#show"
+
+  get "user/:user_id/posts", to: "posts#index"
+  get "user/:user_is/posts/:id", to: "posts#show"
```
Comment (thecodechaser) [inline config/routes.rb]: - [ ] There's typo for the `posts#show` route, Pleaae fix it by replacing `/:user_is/` to `g/:user_id/`

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @shaqayq👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 4 above)
Comment (shaqayq) [reply config/routes.rb]: @thecodechaser !
Thank you for reviewing my code I will change it. :smiley:

**Review by Ismailco — APPROVED** [review]
### Hi @shaqayq, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**